api/endpoints/fake/location_profiles.go

```diff
@@ -0,0 +1,50 @@
+package fake
```

Comment:
Hm I am not a go pro, but these CRUD stubs seems to be repetitive, just changing the type in each of the files.
Isn't there a way to DRY?